pkg/api/conntrack.go

```diff
@@ -37,10 +37,10 @@ type ConnTrack struct {
 }
 
 type ConnTrackOutputRecordTypeEnum struct {
-	NewConnection    string `yaml:"newConnection" doc:"New connection"`
-	EndConnection    string `yaml:"endConnection" doc:"End connection"`
-	UpdateConnection string `yaml:"updateConnection" doc:"Update connection"`
-	FlowLog          string `yaml:"flowLog" doc:"Flow log"`
+	NewConnection string `yaml:"newConnection" doc:"New connection"`
+	EndConnection string `yaml:"endConnection" doc:"End connection"`
+	Heartbeat     string `yaml:"heartbeat" doc:"Heartbeat"`
```

Comment:
Is `Heartbeat` too casual? Is `HeartbeatConnection` preferred?

Reply:
What about `TickConnection` ?